chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -125,8 +125,12 @@ bridge_mappings = physnet1:br-fixed
 # Example: tunnel_types = gre
 # Example: tunnel_types = vxlan
 # Example: tunnel_types = vxlan, gre
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") && @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
Why not. This would look like this:
```ruby
@ml2_type_drivers.select { |t| ["vxlan", "gre"].include?(t) }.join(",")
```

Opinions?

(actually, it'd be slightly different as I'd make this a template variable, so we don't write an empty tunnel_types)